tests/baselines/reference/nodeModulesAllowJsPackageExports(module=node12).errors.txt

```diff
@@ -0,0 +1,107 @@
+tests/cases/conformance/node/allowJs/index.cjs(4,23): error TS1471: Module 'package' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+tests/cases/conformance/node/allowJs/index.cjs(9,23): error TS1471: Module 'inner/mjs' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+tests/cases/conformance/node/allowJs/node_modules/inner/index.d.cts(3,22): error TS1471: Module 'inner/mjs' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+tests/cases/conformance/node/allowJs/node_modules/inner/index.d.ts(2,13): error TS2303: Circular definition of import alias 'cjs'.
+tests/cases/conformance/node/allowJs/node_modules/inner/index.d.ts(3,22): error TS1471: Module 'inner/mjs' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+
+
+==== tests/cases/conformance/node/allowJs/index.js (0 errors) ====
+    // esm format file
+    import * as cjs from "package/cjs";
+    import * as mjs from "package/mjs";
+    import * as type from "package";
+    cjs;
+    mjs;
+    type;
+    import * as cjsi from "inner/cjs";
+    import * as mjsi from "inner/mjs";
+    import * as typei from "inner";
+    cjsi;
+    mjsi;
+    typei;
+==== tests/cases/conformance/node/allowJs/index.mjs (0 errors) ====
+    // esm format file
+    import * as cjs from "package/cjs";
+    import * as mjs from "package/mjs";
+    import * as type from "package";
+    cjs;
+    mjs;
+    type;
+    import * as cjsi from "inner/cjs";
+    import * as mjsi from "inner/mjs";
+    import * as typei from "inner";
+    cjsi;
+    mjsi;
+    typei;
+==== tests/cases/conformance/node/allowJs/index.cjs (3 errors) ====
+    // cjs format file
+    import * as cjs from "package/cjs";
+    import * as mjs from "package/mjs";
+                         ~~~~~~~~~~~~~
+!!! error TS1471: Module 'package/mjs' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+    import * as type from "package";
+                          ~~~~~~~~~
+!!! error TS1471: Module 'package' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.
+    cjs;
+    mjs;
+    type;
+    import * as cjsi from "inner/cjs";
+    import * as mjsi from "inner/mjs";
+                          ~~~~~~~~~~~
+!!! error TS1471: Module 'inner/mjs' cannot be imported using this construct. The specifier only resolves to an es module, which cannot be imported synchronously. Use dynamic import instead.

```

Comment:
'statement' reads better than 'construct' here -- it's more specific and common.

Reply:
I use `construct` because it also applies to dynamic import, which is an expression, and having two error messages just to distinguish statement vs expression seems gratuitous.